proposals/NNNN-Adding-new_parameter_of_Bluetooth_Device_Address.md

```diff
@@ -0,0 +1,116 @@
+            <description>Device OS version</description>
+        </param>
+        <param name="carrier" type="String" minlength="0" maxlength="500" mandatory="false">
+            <description>Device mobile carrier (if applicable)</description>
+        </param>
+        <param name="maxNumberRFCOMMPorts" type="Integer" minvalue="0" maxvalue="100" mandatory="false">
+            <description>Omitted if connected not via BT.</description>
+        </param>         
++       <param name="bluetoothDeviceAddress" type="String"  minlength="0" maxlength="500" mandatory="true">
++           <description>Device BT Address - If cannot get it, set All-F.</description>
++       </param>
+    </struct>
+�c
+    <function name="RegisterAppInterface" functionID="RegisterAppInterfaceID" messagetype="request" since="1.0">
+        <description>
+            Establishes an interface with a mobile application.
+            Before registerAppInterface no other commands will be accepted/executed.
+        </description>
+        
+        <param name="syncMsgVersion" type="SyncMsgVersion" mandatory="true" since="1.0">
+            <description>See SyncMsgVersion</description>
+        </param>
+�c
+        <param name="hashID" type="String" maxlength="100" mandatory="false" since="3.0">
+            <description>
+                ID used to uniquely identify current state of all app data that can persist through connection cycles (e.g. ignition cycles).
+                This registered data (commands, submenus, choie sets, etc.) can be reestablished without needing to explicitly reregister each piece.
+                If omitted, then the previous state of an app's commands, etc. will not be restored.
+                When sending hashID, all RegisterAppInterface parameters should still be provided (e.g. ttsName, etc.).
+            </description>
+        </param>
++       <param name="deviceInfo" type="DeviceInfo" mandatory="true" since="X.X">
+-       <param name="deviceInfo" type="DeviceInfo" mandatory="false" since="3.0">
+            <description>
+                See DeviceInfo.
+            </description>
++           <history>
++                <param name="deviceInfo" type="DeviceInfo" mandatory="false" since="3.0" until="X.X"/>
++           </history>
+        </param>
+        <param name="appID" type="String" maxlength="100" mandatory="true" since="2.0">
+            <description>ID used to validate app with policy table entries</description>
+        </param>
+�c
+    </function>
+```
+
+HMI_API.xml  
+```xml
+<struct name="DeviceInfo">
+    <param name="name" type="String" mandatory="true">
+        <description>The name of the device connected.</description>
+    </param>
+    <param name="id" type="String" mandatory="true">
+        <description>The ID of the device connectedi: either hash of device's USB serial number(in case of USB connection) or has of device's MAC address(in case of BlueTooth or WIFI connection</description>
```

Comment:
```suggestion
        <description>The ID of the device connected; either hash of device's USB serial number (in case of USB connection) or of device's MAC address (in case of BT or WIFI connection</description>
```